### PR TITLE
Class object

### PR DESCRIPTION
**Pull request by shella12**
# Awesome books: refactor to use JavaScript classes

- Modified JScript and added classes to create an object
- Updated CSS in order to achieve the required UI
- Resolved the linter errors
- LocalStorage is added to preserve data

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a readme doc. Kindly include a readme from the template and include relevant information for this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -1,26 +1,75 @@
...
+// };
+// class Book{
+
+// constructor(title=null,author=null){
+// this.title=title;
+// this.author=author;
+
+// }
+
+// }
```
Comment (sja-thedude) [inline script.js:16]: - [ ] Optional: Kindly remove all the comments from this file. Comments make the code dirty.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @shella12 & @skyv26, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (skyv26) [reply script.js:16]: Thanks you so much. Sure We will remove all comments.

**Pull request merged**